scripts/generate_export.py

```diff
@@ -1,12 +1,16 @@
 import os
 import sys
 import time
+from indico.client.client import IndicoClient
 import pandas as pd
 from collections import defaultdict
 from solutions_toolkit.auto_review import Reviewer, FieldConfiguration
 from solutions_toolkit.uipath_block_scripts.config import ExportConfiguration
 from solutions_toolkit.indico_wrapper import IndicoWrapper
-
+from indico.queries import (
```

Comment:
the indico_wrapper object handles this sort of stuff now. Will comment on the rest of these with more details.